ui/static/ui/js/require_config.js

```diff
@@ -49,6 +49,7 @@ var REQUIRE_PATHS = {
   icheckbox: "../ui/js/util/icheckbox.jsx?noext",
   infinite_list: "../ui/js/util/infinite_list.jsx?noext",
   react_overlay_loader: "../ui/js/util/react_overlay_loader.jsx?noext",
+  react_spinner: "../ui/js/util/react_spinner.jsx?noext",
```

Comment:
Nit: let's come up with some nomenclature conventions for files and components, as discussed.